### PR TITLE
Add compound revenue metrics to the API

### PR DESCRIPTION
**Pull request by filips121**
## Changes

Add compound revenue metrics to the API

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] I have tried to find clearer solution before commenting hard-to-understand parts of code
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: lib/sanbase/clickhouse/metric/metric_files/available_v2_metrics.json [hunk 1]
```diff
@@ -4097,6 +4097,86 @@
...
+    "min_interval": "1d",
+    "table": "daily_metrics_v2",
+    "has_incomplete_data": true,
+    "data_type": "timeseries",
+    "docs_links": ["https://academy.santiment.net/metrics/lending-and-borrowing-protocols/compound"]
+  },
+  {
+    "human_readable_name": "Daily Compound Total Revenue",
+    "name": "compound_total_revenue_usd",
+    "metric": "compound_total_revenue_usd",
```
Comment (IvanIvanoff) [inline lib/sanbase/clickhouse/metric/metric_files/available_v2_metrics.json:4143]:
```suggestion
    "human_readable_name": "Total Compound Total Revenue",
    "name": "compound_total_revenue_usd",
    "metric": "compound_total_revenue_usd",
```

File: lib/sanbase/clickhouse/metric/metric_files/available_v2_metrics.json [hunk 2]
```diff
@@ -4097,6 +4097,86 @@
...
+    "min_interval": "1d",
+    "table": "daily_metrics_v2",
+    "has_incomplete_data": true,
+    "data_type": "timeseries",
+    "docs_links": ["https://academy.santiment.net/metrics/lending-and-borrowing-protocols/compound"]
+  },
+  {
+    "human_readable_name": "Daily Compound Revenue in USD",
+    "name": "compound_revenue_usd",
+    "metric": "compound_revenue_usd",
```
Comment (IvanIvanoff) [inline lib/sanbase/clickhouse/metric/metric_files/available_v2_metrics.json:4123]: If this is daily, maybe expose the metric as `daily_compound_revenue_usd`?

File: lib/sanbase/clickhouse/metric/metric_files/available_v2_metrics.json [hunk 3]
```diff
@@ -4097,6 +4097,86 @@
...
+    },
+    "aggregation": "sum",
+    "min_interval": "1d",
+    "table": "daily_metrics_v2",
+    "has_incomplete_data": true,
+    "data_type": "timeseries",
+    "docs_links": ["https://academy.santiment.net/metrics/lending-and-borrowing-protocols/compound"]
+  },
+  {
+    "human_readable_name": "Daily Compound Total Cumulative Revenue",
```
Comment (IvanIvanoff) [inline lib/sanbase/clickhouse/metric/metric_files/available_v2_metrics.json:4161]: I am getting confused by having `daily` and `total` at the same time. What is the meaning exactly?
`Daily` refers to the computed interval (i.e. 1 value per day) and `Total` means that the total cumulative revenue is obtained?

File: lib/sanbase/clickhouse/metric/metric_files/available_v2_metrics.json (hunk 2 above)
Comment (filips121) [reply lib/sanbase/clickhouse/metric/metric_files/available_v2_metrics.json:4123]: Do you mean to add the `daily` prefix to all revenue metrics:
- `daily_compound_revenue`
- `daily_compound_revenue_usd`
- `daily_compound_total_revenue_usd`
- `daily_compound_total_cumulative_revenue_usd`

File: lib/sanbase/clickhouse/metric/metric_files/available_v2_metrics.json (hunk 3 above)
Comment (filips121) [reply lib/sanbase/clickhouse/metric/metric_files/available_v2_metrics.json:4161]: Total means all assets combined in usd for this compound protocol

File: lib/sanbase/clickhouse/metric/metric_files/available_v2_metrics.json (hunk 2 above)
Comment (IvanIvanoff) [reply lib/sanbase/clickhouse/metric/metric_files/available_v2_metrics.json:4123]: After discussing in DM, this proposal is retracted.

**Pull request merged**